website/docs/guides/version-6-upgrade.html.markdown

```diff
@@ -136,6 +141,18 @@ This is not recommended.
 
 `tags_all` has been removed.
 
+## data-source/aws_ecs_task_definition
+
+Remove `inference_accelerator` from your configuration—it no longer exists. Amazon Elastic Inference reached end of life in April 2024.
```

Comment:
Perhaps rephrase slightly to align with the other existing removal entries?

```suggestion
`inference_accelerator` has been removed. Amazon Elastic Inference reached end of life in April 2024.
```

Reply:
I think the guide should be telling people what they need to do, imperative language: "remove this, do that, update this"

Reply:
Seems reasonable to me, though most entries are not currently doing that. Perhaps we just need to plan for standardizing at the end so each section is consistent.